modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -30,15 +54,26 @@ def va_profile=(new_va_profile)
       super(new_va_profile)
     end
 
+    def loa3_user
+      loa3?
```

Comment:
We may need to consider the concept of LoA 3 or higher or LoA 2 or higher...

Reply:
This is an mvi requirement

Reply:
I guess what I'm trying to say is that we should want to use veteran for more than just AVI call so having the concept of and or greater and or lesser would allow us to reuse this model for more than just the MVI, for example in the case where EVSS needs only 2 we should allow for that etc.

Reply:
However if we don’t have loa3 we can’t get the missing information needed for the. Evss headers. So i guess we can require them to be passed vs obtained from mvi but without them the evss call will fail